src/util/types.ts

```diff
@@ -1103,6 +1103,8 @@ export interface LabelOption extends TextCommonOption {
     // formatter?: string | ((params: CallbackDataParams) => string)
 
     rich?: Dictionary<TextCommonOption>
+    ellipsis?: TextStyleProps['ellipsis']
```

Comment:
`ellipsis` and `lineOverflow` are not supported in the labels. We can't add them to general `LabelOption`